README.md

```diff
@@ -49,15 +49,42 @@ The json must have the following fields:
 This file is required for the coverage integration so it needs to be added
 to the repository as well.
 
-1. Create a new pipeline definition in Buildkite. For this step ask one of the
-rust-vmm Buildkite [admins](CODEOWNERS) to create one for you. Add a pipeline
-step that is uploading the rust-vmm-ci pipeline:
+3. Create a new pipeline definition in Buildkite. For this step ask one of the
+rust-vmm Buildkite [admins](CODEOWNERS) to create one for you.
 
+4. There is a script that autogenerates a dynamic Buildkite pipeline. To run 
+the CI using this dynamic pipeline, you need to add a step that is uploading
+the rust-vmm-ci pipeline:
 ```bash
-buildkite-agent pipeline upload rust-vmm-ci/.buildkite/pipeline.yml
+./rust-vmm-ci/.buildkite/autogenerate_pipeline.py | buildkite-agent pipeline upload
+```
+This allows overriding some values and extending others through environment 
+variables. 
+- `X86_LINUX_AGENT_TAGS`: overrides the tags by which the x86_64 linux agent is
```

Comment:
Let's also write about the defaults and say that for most use cases overriding these is not necessary.

We might also want to give a few examples when we do want to override them (platform needs a custom device that is not available on the existing test instances, need for a specialized hypervisor - for example, microsoft hyper-v).

Reply:
Done.